src/postgraphql/schema/exportPostGraphQLSchema.ts

```diff
@@ -0,0 +1,26 @@
+import { writeFileSync } from 'fs'
+import { graphql, GraphQLSchema } from 'graphql'
+import { introspectionQuery, printSchema } from 'graphql/utilities'
+import { graphql } from 'graphql'
+
+/**
+ * Exports a PostGraphQL schema by looking at a Postgres client.
+ */
+export default async function exportPostGraphQLSchema (
+  schema: GraphQLSchema,
+  options: {
+    exportPathJSON?: string,
+    exportPathSchema?: string,
+  } = {},
+): Promise<void> {
+  // JSON version
+  if (options.exportPathJSON) {
+    const result = await graphql(schema, introspectionQuery)
+    writeFileSync(options.exportPathJSON, JSON.stringify(result))
```

Comment:
Could we make this async? We don’t need another library or utility function, we could just do it inline. It’s a little bit of non-DRY code, but I think it’s worth it:

```js
await new Promise((resolve, reject) => {
  writeFile(options.exportPathJSON, JSON.stringify(result), error => {
    if (error) reject(error)
    else resolve()
  })
})
```

Same thing below for `options.exportPathSchema`.